create-maint-branch.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e
+
+git checkout -b "$BRANCH"
+
+yq e -i '.github.organisations[].repositories[].branch = env(BRANCH)' .updatebot.yml
+
+yq e -i '.on.push.branches = [ env(BRANCH) ]' .github/workflows/main.yml
+yq e -i '.on.pull_request.branches = [ env(BRANCH) ]' .github/workflows/main.yml
+
+readonly NEXT_VERSION=${VERSION%\-SNAPSHOT}
+yq -i e '(.jobs.build.steps[] |  select(.id == "next-release") | .with.next-version) |=  env(NEXT_VERSION) ' .github/workflows/main.yml
+
+mvn -B versions:set -DnewVersion="$VERSION" -DprocessAllModules=true -DgenerateBackupPoms=false
+
```

Comment:
I think it might be a good idea to print the diff here for [more clarity in the output ](https://app.travis-ci.com/github/Alfresco/alfresco-community-repo/jobs/566267638#L522)like [Repo does](https://github.com/Alfresco/alfresco-community-repo/blob/master/scripts/travis/update_downstream.sh#L32-L33):

```
git status
git --no-pager diff pom.xml
```

![image](https://user-images.githubusercontent.com/11974750/162245927-b50b4a67-6bd7-4843-a728-b8db847565f2.png)

Reply:
It's a good idea. I added the status and diff.